### PR TITLE
Add a custom Kairos image

### PR DESCRIPTION
**Pull request by sdwilsh**
Right now, this just adds `vim`, but we can do much more with it over time!  This also supports amd64 and Raspberry Pi systems that support arm64.

### CODE REVIEW

File: .github/workflows/kairos-ubuntu-22-lts.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: earthly/actions-setup@v1.0.8
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          version: "0.7.1"
```
Comment (robarnold) [inline .github/workflows/kairos-ubuntu-22-lts.yml:27]: We'll want to make sure renovate can update this version of earthly

File: .github/workflows/kairos-ubuntu-22-lts.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v2
```
Comment (robarnold) [inline .github/workflows/kairos-ubuntu-22-lts.yml:29]: Is this needed? I thought we could use the USB image build workflow as a base. Same with the docker login.

File: .github/workflows/kairos-ubuntu-22-lts.yml (hunk 1 above)
Comment (sdwilsh) [reply .github/workflows/kairos-ubuntu-22-lts.yml:27]: The existing GitHub workflow also has this issue; let's file an issue and fix both?

File: .github/workflows/kairos-ubuntu-22-lts.yml (hunk 2 above)
Comment (sdwilsh) [reply .github/workflows/kairos-ubuntu-22-lts.yml:29]: qemu is needed in order to build for arm

**Review by robarnold — APPROVED** [review]

**Comment by robarnold** [comment]
What instructions do we have for migrating our installs?

**Comment by sdwilsh** [comment]
> What instructions do we have for migrating our installs?

We need to pass in a source image instead, I think: https://kairos.io/docs/upgrade/manual/

**Pull request merged**